### PR TITLE
Add extra population to historic population

### PR DESCRIPTION
**Pull request by seb231**
Combine the historic population with the extra population due to population growth.

### CODE REVIEW

**Comment by seb231** [comment]
lgtm

File: test/witan/send/send_test.clj [hunk 1]
```diff
@@ -73,10 +73,33 @@
...
-               :id [5 5 6 6 7 7 8 8 9 9 10 10 11 11 12 12 13 13 14 14 15 15 16 16
-                    17 17 18 18 19 19 20 20 21 21 22 22 23 23 24 24]
                :sim-num [1 2 1 2 1 2 1 2 1 2 1 2 1 2 1 2 1 2 1 2 1 2 1 2 1 2 1 2 1 2
-                         1 2 1 2 1 2 1 2 1 2]}))
+                         1 2 1 2 1 2 1 2 1 2]
+               :id [5 5 6 6 7 7 8 8 9 9 10 10 11 11 12 12 13 13 14 14 15 15 16 16
+                    17 17 18 18 19 19 20 20 21 21 22 22 23 23 24 24]}))
+
+(def population-total
+  (ds/dataset {:year [2014 2014 2014 2014 2014 2014 2016 2016
```
Comment (acron0) [inline test/witan/send/send_test.clj]: This scares me a lot. How come this stuff isn't being loaded from a CSV?

Comment (seb231) [inline test/witan/send/send_test.clj]: Possible, but would require us to make the file, then (I think) load it and run a number of other functions on it. What precisely is bad about it?

Comment (acron0) [inline test/witan/send/send_test.clj]: Having columns laid out like this makes it almost impossible to parse a single row, which obviously makes changes hairy and fragile. As a CSV it'd be way easier to grok. Also, in other projects we've loaded CSVs.

**Comment by seb231** [comment]
After discussion with @acron0 :shipit:

**Pull request merged**